### PR TITLE
New version: Microsoft.DevHome version 0.503.261.0

### PR DESCRIPTION
**Pull request by cinnamon-msft**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120403)

### CODE REVIEW

**Review by mdanish-kh — APPROVED** [review]

**Comment by Trenly** [comment]
From the AppxManifest
```xml
		<b5:Package Type="application" Version="0.502.261.0" Architecture="arm64" FileName="AppxMetadata\Stub\DevHomeStubPackage_0.502.261.0_ARM64.msix" Offset="89" Size="929513" IsStub="true">
			<b5:Resources>
				<b5:Resource Language="EN-US"/>
				<b5:Resource Scale="200"/>
			</b5:Resources>
			<b4:Dependencies>
				<b4:TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.22621.0" MaxVersionTested="10.0.22621.0"/>
			</b4:Dependencies>
		</b5:Package>
```

Looks like the GitHub tag and the package name are incorrect

File: manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.503.261.0
```
Comment (Trenly) [inline manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml:5]:
```suggestion
PackageVersion: 0.502.261.0
```
The folder will need to be renamed after changing this value

File: manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.503.261.0
```
Comment (Trenly) [inline manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.502.261.0
```
The folder will need to be renamed after changing this value

File: manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.503.261.0
```
Comment (Trenly) [inline manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.yaml:5]:
```suggestion
PackageVersion: 0.502.261.0
```
The folder will need to be renamed after changing this value

File: manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.503.261.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.22000.0
```
Comment (Trenly) [inline manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml:8]:
```suggestion
MinimumOSVersion: 10.0.22621.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
@Trenly - I think this is a case where the stub package included is also getting validated in the pipelines. Similar case in previous closed PR https://github.com/microsoft/winget-pkgs/pull/112622.

- Related to https://github.com/microsoft/winget-cli/issues/3452 (not sure if this PR made it into the pipelines)

The version from ARP seems to be correct

image

**Comment by Trenly** [comment]
> @Trenly - I think this is a case where the stub package included is also getting validated in the pipelines.

Ah, I think you’re right

File: manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml (hunk 4 above)
Comment (mdanish-kh) [reply manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml:8]: From AppxBundleManifest.xml

```xml
		<Package Type="application" Version="0.503.261.0" Architecture="x86" FileName="DevHome-x86.msix" Offset="278577046" Size="129502986">
			<Resources>
				<Resource Language="EN-US"/>
				<Resource Language="DE-DE"/>
				<Resource Language="ES-ES"/>
				<Resource Language="FR-FR"/>
				<Resource Language="IT-IT"/>
				<Resource Language="JA-JP"/>
				<Resource Language="KO-KR"/>
				<Resource Language="PT-BR"/>
				<Resource Language="QPS-PLOC"/>
				<Resource Language="QPS-PLOCA"/>
				<Resource Language="QPS-PLOCM"/>
				<Resource Language="RU-RU"/>
				<Resource Language="ZH-CN"/>
				<Resource Language="ZH-TW"/>
			</Resources>
			<b4:Dependencies>
				<b4:TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.22000.0" MaxVersionTested="10.0.22000.0"/>
			</b4:Dependencies>
		</Package>
		<Package Type="application" Version="0.503.261.0" Architecture="x64" FileName="DevHome-x64.msix" Offset="138258783" Size="140318193">
			<Resources>
				<Resource Language="EN-US"/>
				<Resource Language="DE-DE"/>
				<Resource Language="ES-ES"/>
				<Resource Language="FR-FR"/>
				<Resource Language="IT-IT"/>
				<Resource Language="JA-JP"/>
				<Resource Language="KO-KR"/>
				<Resource Language="PT-BR"/>
				<Resource Language="QPS-PLOC"/>
				<Resource Language="QPS-PLOCA"/>
				<Resource Language="QPS-PLOCM"/>
				<Resource Language="RU-RU"/>
				<Resource Language="ZH-CN"/>
				<Resource Language="ZH-TW"/>
			</Resources>
			<b4:Dependencies>
				<b4:TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.22000.0" MaxVersionTested="10.0.22000.0"/>
			</b4:Dependencies>
		</Package>
		<Package Type="application" Version="0.503.261.0" Architecture="arm64" FileName="DevHome-arm64.msix" Offset="3703639" Size="134555074">
			<Resources>
				<Resource Language="EN-US"/>
				<Resource Language="DE-DE"/>
				<Resource Language="ES-ES"/>
				<Resource Language="FR-FR"/>
				<Resource Language="IT-IT"/>
				<Resource Language="JA-JP"/>
				<Resource Language="KO-KR"/>
				<Resource Language="PT-BR"/>
				<Resource Language="QPS-PLOC"/>
				<Resource Language="QPS-PLOCA"/>
				<Resource Language="QPS-PLOCM"/>
				<Resource Language="RU-RU"/>
				<Resource Language="ZH-CN"/>
				<Resource Language="ZH-TW"/>
			</Resources>
			<b4:Dependencies>
				<b4:TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.22000.0" MaxVersionTested="10.0.22000.0"/>
			</b4:Dependencies>
		</Package>
```

For non-stub packages, I believe the MinOS version is `10.0.22000.0`

**Comment by AmelBawa-msft** [comment]
@mdanish-kh, that's correct. The PR you linked should resolve the issue. I believe the pipeline is still using an older nuget which does not include that fix. We should do that hopefully by next Dev Home release to avoid the same issue.

**Review by denelon — APPROVED** [review]

**Pull request merged**